data/incipias/incipias outfits.txt

```diff
@@ -0,0 +1,279 @@
+		"velocity" 1
+		acceleration 5
+		drag .2
+		"lifetime" 120
+		"shield damage" 410
+		"hull damage" 430
+		"homing" 1
+		"optical tracking" 1
+		"turn" 3
+
+effect "star tail hit"
+	sprite "effect/star tail hit"
+		"no repeat"
+		"frame rate" 40
+	sound "star tail hit"
+	"lifetime" 30
+	"velocity scale" 0.
+
+effect "star tail live"
+	sprite "effect/star tail live"
+		"no repeat"
+		"frame rate" 28
+	"lifetime" 15
+	"velocity scale" 0.
+
+outfit "Star Tail Cell"
+	category "Ammunition"
+	cost 500
+	thumbnail "outfit/star tail cell"
+	"mass" .1
+	"star tail capacity" -1
+	description "A very dense gas contained in a small cell. The amounts of energy it can free are exceptional."
+
+outfit "Star Tail Storage"
+	category "Ammunition"
+	cost 500
+	thumbnail "outfit/star tail cell"
+	"mass" 10
+	"star tail capacity" 20
+	description "One layer of this easily expandable system is already shipped with the gun itself. More layers are just stacked on top."
+
+# Shields/Energy:
+outfit "Gas-Class Shield"
+	category "Systems"
+	thumbnail "outfit/gas class shield"
+	mass 60
+	"outfit space" -60
+	"shield generation" .1
+	"energy generation" 4.2
+	"heat generation" 5
+	description "Shield technology is one of the most recent developments of the Incipias. While this shield appears similar to a human shield device, it works on a completely different base. In its core hydrogen is burned that provides energy for regeneration, but because scientists saw a more efficient use of this burning technology, the Incipias also made it the main power source of all of their ships."
+
+outfit "Liquid-Class Shield"
+	category "Systems"
+	thumbnail "outfit/liquid class shield"
+	mass 80
+	"outfit space" -90
+	"shield generation" .25
+	"energy generation" 6.1
+	"heat generation" 6
+	description "This more advanced shield generator burns a liquid instead of a gas. This raises the efficiency considerably but also requires some extra structures, making this outfit popular only on larger ships."
+
+outfit "Solid-Class Shield"
+	category "Systems"
+	mass 100
+	"outfit space" -120
+	"shield generation" .45
+	"energy generation" 6
+	"heat generation" 8
+	description "This shield generator is the most efficient available to the Incipias. It burns its input in stages from solid in the center to a gaseous state in the outer hull. Sadly this efficiency requires a lot of adjusting systems to stabilize the process, and therefore only the heaviest warships make use of the Solid-Class Shield."
+
+outfit "Liquid Hydrogen Cell"
+	category "Power"
+	thumbnail "outfit/liquid hydrogen cell"
+	mass 20
+	"outfit space" -20
+	"energy capacity" 500
+	description "This form of energy storage was discovered early by the Incipias because of the great amount of hydrogen on their homeworld. Because Incipias ships mostly access their energy at the time it is produced, only a small amount of storage is needed. Most of their ships have only one cell installed as a consequence."
+
+# Engines:
```

Comment:
These engines are all missing outfit and engine space requirements.

Reply:
That's because I still have to think about their size, I am not sure if they should be rather big or small. Suggestions?

Reply:
I'm not sure about the smaller ones (the green-class and blue-class seem to have the exact same stats on the thrusters, so I'm guessing those aren't finished yet?), but since the engine space of the Venta is 130, you probably want the size of the Red-class thruster and steering to add up to about 130 or slightly less. That'd put them in a similar size category to Korath planetary engines (120 tons total), or human X4 series ion engines (138 tons total).